packages/website/src/images/index.ts

```diff
@@ -1,32 +1,34 @@
+/* eslint @typescript-eslint/no-var-requires: 0 */
```

Comment:
eslint wants us to use `import` here, but I couldn't get it to work. If we can, then we should go ahead and re-enable this rule, and change the `require`'s to `import`'s